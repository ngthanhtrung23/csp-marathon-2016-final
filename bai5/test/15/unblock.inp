1222.3
1444.3
00...3
.556..
..7688
9976..
