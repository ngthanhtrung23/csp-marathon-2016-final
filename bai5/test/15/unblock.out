21
0 R 1
1 D 2
2 L 1
4 L 1
6 U 3
5 R 2
7 U 1
9 R 3
1 D 2
0 L 1
7 D 1
5 L 3
7 U 2
8 L 3
3 D 3
9 L 2
6 D 3
2 R 3
4 R 3
7 U 2
0 R 4

